### PR TITLE
T/147: If there is not tag in the repo release:dependencies throws

### LINKED ISSUE #147: If there is not tag in the repo release:dependencies throws
```
(master c80166f) p@m /workspace/ckeditor5> gulp release:dependencies
[18:37:35] Using gulpfile /workspace/ckeditor5/gulpfile.js
[18:37:35] Starting 'release:dependencies'...
[18:37:36] Error while executing git describe --abbrev=0 --tags 2> /dev/null:
```

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: A util `getPackagesToRelease()` will not break the release process when some package is released for the first time. Closes #147.

### CODE REVIEW

**Comment by pomek** [comment]
I need to test this change manually. Please give me few mins.

**Comment by pomek** [comment]
Works fine!

image

image

https://github.com/pomek/ckeditor5-d/releases/tag/v0.0.2

File: packages/ckeditor5-dev-env/tests/release-tools/utils/getpackagestorelease.js [hunk 1]
```diff
@@ -192,5 +192,43 @@ describe( 'dev-env/release-tools/utils', () => {
 					expect( packages.get( '@ckeditor/ckeditor5-utils' ) ).to.equal( undefined );
 				} );
 		} );
+
+		it( 'does not throw when packages are released for the first time', () => {
```
Comment (Reinmar) [inline packages/ckeditor5-dev-env/tests/release-tools/utils/getpackagestorelease.js]: This test does not throw without the fix. So. what is it checking?

**Comment by Reinmar** [comment]
Missing an integration test. One of the new tests doesn't check this case and the other is very low level. How do you know that the code really works?

**Comment by pomek** [comment]
As you reported in a ticket:

> ```
> [18:37:36] Error while executing git describe --abbrev=0 --tags 2> /dev/null:
> ```

This error was thrown by one of the utils (`version.js`) so I fixed it and `getPackagesToRelease()` will not break the release if tag will not exist.

**Comment by Reinmar** [comment]
How do you know that something else won't break? Perhaps you tested it, but I don't want to create a new repo just to test it and repeat that N times in the future every time we'll change something in this code.

Unit tests may verify a piece of logic. Integration tests verify features. I'd like this feature to be tested well because it's a complex piece of code and we had issues in it.

**Comment by pomek** [comment]
Will be fixed in #216 because it delivers an integration test.

**Pull request closed without merging**